go/chat/unfurl/scrape_generic.go

```diff
@@ -9,58 +9,198 @@ import (
 	"github.com/keybase/client/go/protocol/chat1"
 )
 
+// Contents are scored based on source. Higher scores win but falsey values
+// always loose.
+const (
+	defaultScore          = 1
+	defaultOpenGraphScore = 10
```

Comment:
Also I think OpenGraph should win over Twitter, they are more likely to be generally applicable.

Reply:
Sorry one other comment, I think these scores should also be per attribute from the generic unfurl for maximum flexibility. They can "inherit" from this default one some way, but I thought be able to tweak the score for title separately from image.

Reply:
these technically win now since first writer wins for equal scores but i can bump open graph to slightly higher. for per attribute i think that should be determined within the switch statement closest to the processing logic for that tag. if that gets unwieldy we can make a map with comments, but i think it's more likely to be domain not tag specific

Reply:
Let’s talk about it when I get in